hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -972,6 +972,17 @@
   /** Whether nonces are enabled; default is true. */
   public static final String HBASE_RS_NONCES_ENABLED = "hbase.regionserver.nonces.enabled";
 
+  /** Some configuration param about multiget in parallel */
+  public static final String HBASE_RS_PARALLEL_GET_ENABLED =
+      "hbase.regionserver.parallel.get.enabled";
```

Comment:
It's better to add the new configurations to hbase-default.xml and hbase-default.adoc.

Reply:
make sense